src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -615,6 +615,8 @@ internal static bool IsBuildOnlyDiagnostic(ErrorCode code)
                 case ErrorCode.ERR_InterceptorArityNotCompatible:
                 case ErrorCode.ERR_InterceptorCannotBeGeneric:
                 case ErrorCode.ERR_InterceptableMethodMustBeOrdinary:
+                case ErrorCode.ERR_PossibleAsyncIteratorWithoutYield:
+                case ErrorCode.ERR_PossibleAsyncIteratorWithoutYieldOrAwait:
```

Comment:
How is `IsBuildOnlyDiagnostic()` used? In short, what is the effect of returning `true` for these error codes?

Reply:
It's used by the ide for live diagnostic filtering. It will mean that the ide won't suppress the warning from build when the compiler doesn't report it.